libs/api/src/lib/graphql/permissions.ts

```diff
@@ -708,7 +708,6 @@ export const EditorPermissions: Permission[] = [
   CanGetPeers,
   CanGetPeerProfile,
   CanLoginEditor,
-  CanGetSettings,
```

Comment:
I think we should still keep this permission in the api. Just remove the qualifying permission of "CAN_GET_SETTINGS" from the settings page .

Reply:
Ah you're right. This makes sense.